appsec/src/extension/helper_process.c

```diff
@@ -522,8 +524,8 @@ static char **nullable _split_params(
     }
 
     // we never write more than the original size of the params
-    char *params_buffer = emalloc(strlen(orig_params_str) + 1);
-    char *wp = params_buffer; // write pointer
+    char *params_buffer = emalloc(strlen(orig_params_str) + 1); // NOLINT
+    char *wp = params_buffer;                                   // write pointer
```

Comment:
Wrong linting

Reply:
What do you mean?

Reply:
Line 528 is aligning the comment with the one on the previous line. However the one on the following is not. So I'd say, it should not be aligned

Reply:
clang-format quirk I suppose